testgen/generators.go

```diff
@@ -21,6 +20,7 @@ import (
 	"github.com/ethereum/go-ethereum/rpc"
 	"github.com/holiman/uint256"
 	"golang.org/x/exp/maps"
+	"golang.org/x/exp/slices"
```

Comment:
I don't think we need to change to `exp/slices`. "slices" package has been part of std since go 1.21.

Reply:
I modified this to use slices